decs.d.ts

```diff
@@ -15,4 +20,104 @@ declare module '@gridsuite/commons-ui' {
     }
 
     export function useSnackMessage(): UseSnackMessageReturn;
+
+    type Input = string | number;
+    type Options = Array<{
+        id: string;
+        label: string;
+    }>;
+
+    interface AutocompleteInputProps
+        extends Omit<
+            AutocompleteProps<
+                string,
+                boolean | undefined,
+                boolean | undefined,
+                boolean | undefined
+            >,
+            'renderInput' // we already defined it in our custom Autocomplete
+        > {
+        name: string;
+        label?: string;
+        options: Array<any>; // https://mui.com/material-ui/react-autocomplete/#options-structure
+        outputTransform?: (value: string) => string;
+        inputTransform?: (value: string) => string;
+        readOnly?: boolean;
+        previousValue?: string;
+        allowNewValue?: boolean;
+        onChangeCallback?: () => void;
+        formProps?: TextFieldProps;
+    }
+
+    export function AutocompleteInput(props: AutocompleteInputProps);
```

Comment:
```suggestion
    export function AutocompleteInput(props: AutocompleteInputProps): ReactNode;
```

same for all exports, otherwise it returns any

Reply:
Ok I will type the return value.
But they are all ReactElement, why using a wider type ?